MetadataExtractor/IO/BufferReader.Indexed.cs

```diff
@@ -178,6 +177,9 @@ namespace MetadataExtractor.IO;
 
     public readonly string GetString(int index, int bytesRequested, Encoding encoding)
     {
+        if (bytesRequested < 0)
+            throw new ArgumentOutOfRangeException(nameof(bytesRequested), "Must be 0 or greater");
```

Comment:
super nit: exception messages should end with a full stop in .NET.

Reply:
noted and fixed!